packages/tasklist-client-node-js/src/lib/TasklistRESTClient.ts

```diff
@@ -108,7 +108,7 @@ export class TasklistRESTClient {
      * @param formId
      * @param processDefinitionKey
      */
-    public async getForm(formId: string, processDefinitionKey: string): Promise<{ form: Form }> {
+    public async getForm(formId: string, processDefinitionKey: string): Promise<Form> {
```

Comment:
This was one of the bugs, it was causing the `getForm` spec to fail. 

GraphQL was returning the `Form` shape as a `form` property on the response object; the REST endpoint returns the `Form` shape flattened on the response object.